### PR TITLE
3.6/dev intrepid lwright

### PR DESCRIPTION
**Pull request by LoganWright**
#726 
#729 
#721 

Replaces original PR: #733

### CODE REVIEW

File: Modules/Dining/MITDiningHouseMealListViewController.m [hunk 1]
```diff
@@ -152,9 +157,17 @@ - (void)updateCurrentlyDisplayedItems
             }
         }
         self.currentlyDisplayedItems = filteredItems;
-        [self updateNotificationLabel];
     }
+    
+    if (self.currentlyDisplayedItems.count > 0) {
+        self.tableView.tableFooterView = nil;
+    } else {
+        self.tableView.tableFooterView = [UIView new];
```
Comment (rosslebeau) [inline Modules/Dining/MITDiningHouseMealListViewController.m]: What's the deal with this?

Comment (LoganWright) [inline Modules/Dining/MITDiningHouseMealListViewController.m]: When there's meals, they want the normal tableview behavior where it shows empty cells.  If there's no meal, or no items, they want the tableview to be blank.  This makes those cells not show up.

Comment (rosslebeau) [inline Modules/Dining/MITDiningHouseMealListViewController.m]: "Those cells" == which cells?

How is the display message being shown? A label somewhere?

Comment (LoganWright) [inline Modules/Dining/MITDiningHouseMealListViewController.m]: Added line notes explaining the behavior.

**Comment by rosslebeau** [comment]
:+1:

**Pull request merged**